about.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Plug about</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <section class="start">
+        <header class="realHead">
+            <div class="navLine">
+                <nav>
+                    <div class="sideMback">
+                        <img class="sideMenu" onclick="openMobileMenu()" src="img/sideMenu.png" alt="">
+                    </div>
+                    <div class="navSlide">
+                        <div class="mediNbac">
+                            <ul class="mediaNav">
+                                <li><img src="img/facebook.png" alt="" class="naVimg"></li>
+                                <li><img src="img/twitter.png" alt="" class="naVimg"></li>
+                                <li>English</li>
+                                <li>My page</li>
+                                <li>Login</li>
+                            </ul>
+                        </div>
+                        <div class="relNav">
+                            <div>
+                                <a href="index.html"><ul class="navle">
+                                    <li><img class="logoN" src="img/plug_icon.png" alt=""></li>
+                                    <li>
+                                        <h1>MyPlug</h1>
+                                    </li>
+                                </ul></a>
+                            </div>
+                            <div>
+                                <ul class="navRig">
+                                    <li><h2><a href="index.html" class="menuli">Home</a></h2></li>
+                                    <li><h2><a href="about.html" class="menuli">About</a></h2></li>
+                                    <li><h2><a href="#ph" class="menuli">Intro</a></h2></li>
+                                    <li><h2><a href="#We_do" class="menuli">Create</a></h2></li>
+                                    <li><h2><a href="#tea" class="menuli">Team</a></h2></li>
+                                    <li><h2><a href="#pat" class="menuli">Partners</a></h2></li>
+                                </ul>
```

Comment:
- [ ] Optional suggestion

- I would recommend changing the font size of your menu to look great.

![image3](https://user-images.githubusercontent.com/55635977/142716729-3912b51a-bcbc-43f1-987c-9ed9bd35381f.png)